compute/address/address_test.go

```diff
@@ -31,6 +33,107 @@ import (
 )
 
 // helper functions
+
+func createTestInstance(projectID, zone, instanceName string) error {
+	// projectID := "your_project_id"
+	// zone := "europe-central2-b"
+	// instanceName := "your_instance_name"
+
+	ctx := context.Background()
+	instancesClient, err := compute.NewInstancesRESTClient(ctx)
+	if err != nil {
+		return fmt.Errorf("NewInstancesRESTClient: %w", err)
+	}
+	defer instancesClient.Close()
+
+	imagesClient, err := compute.NewImagesRESTClient(ctx)
+	if err != nil {
+		return fmt.Errorf("NewImagesRESTClient: %w", err)
+	}
+	defer imagesClient.Close()
+
+	// List of public operating system (OS) images: https://cloud.google.com/compute/docs/images/os-details.
+	newestDebianReq := &computepb.GetFromFamilyImageRequest{
+		Project: "debian-cloud",
+		Family:  "debian-10",
```

Comment:
```suggestion
		Family:  "debian-12",
```

according to the [image family docs](https://cloud.google.com/compute/docs/images/os-details#debian) the debian-10 family is due for deprecation this month.